src/Application.vala

```diff
@@ -413,12 +421,8 @@ namespace Switchboard {
                 deck.transition_duration = 200;
                 deck.visible_child = category_view;
             } else {
-                if (previous_plugs.size > 0) {
-                    load_plug (previous_plugs.@get (0));
-                    previous_plugs.remove_at (0);
-                } else {
-                    switch_to_plug (current_plug);
-                }
+                load_plug (previous_plugs.@get (0));
+                previous_plugs.remove_at (0);
```

Comment:
`previous_plugs` is removed by PR #196  so this will fail if that PR is merged.

Reply:
That's `loaded_plugs` which is a separate list

Reply:
Oops - ignore that.